engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetWireframe.java

```diff
@@ -56,12 +56,16 @@ public RenderPipelineTask generateTask() {
     @Override
     public boolean isEqualTo(StateChange stateChange) {
         if (stateChange instanceof SetWireframe) {
-            return this.enabled == ((SetWireframe) stateChange).enabled;
+            return this.enabled == ((SetWireframe) stateChange).isEnabled();
```

Comment:
On second thought: can't you use that quirk of java that allows instances of the same class to access each other private members? Or did I perhaps explicitly suggest to not use it? I think I did, didn't I? :smile: